src/components/shared/input-style/theme.ts

```diff
@@ -1,8 +0,0 @@
-import theme from 'src/styles/theme';
```

Comment:
Could you explain, please, why did you delete this?

Reply:
![image](https://user-images.githubusercontent.com/41474876/154563101-ef6415ae-8fa1-4532-b4f7-522171296e25.png)